Dockerfile

```diff
@@ -1,9 +1,7 @@
 # Copyright Contributors to the Open Cluster Management project
 
-FROM registry.access.redhat.com/ubi8/nodejs-14:1 as builder
-
-USER root
-RUN yum install git -y
+FROM registry.ci.openshift.org/open-cluster-management/common-nodejs-parent:nodejs-14
+RUN microdnf install git -y
```

Comment:
The `make` targets below this rely on NPM, so I think we'll need to add:
```
RUN ./install-npm.sh
```
https://github.com/open-cluster-management/common-nodejs-parent/blob/6b20341a3d71efdb5e541a2b2b2727a6dcfb71c7/Dockerfile#L11-L14